src/edu/csus/ecs/pc2/core/execute/SandboxResults.java

```diff
@@ -7,14 +7,20 @@
 import java.util.Properties;
 
 /**
- * Sandbox Results loader
+ * Sandbox Results loader.
  * 
```

Comment:
The comment indicates that this class is a "loader"... yet the NAME of the class implies that it is an encapsulation of "results".  At a minimum, the header comments should provide enough additional information to make it clear why this is a "loader" when it appears to be a "Results".  Better, the class name should be changed to match the class function if necessary.

Reply:
not doing doc at this time.   Once a design has been decided on will do doc work.
This PR is not intended to be merged to develop as described in a previous email.